### PR TITLE
Add github link

### PR DESCRIPTION
**Pull request by vladimirdotk**

### CODE REVIEW

File: components/Footer/Footer.css [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 .footer-menu__item {
     display: inline-block;
-    margin-right: 24px;
+    margin-right: 15px;
```
Comment (binakot) [inline components/Footer/Footer.css]: Why did you change this value?

Comment (vladimirdotk) [reply components/Footer/Footer.css]: to keep all of them in one line

Comment (binakot) [reply components/Footer/Footer.css]: I just checked that. You're right!
